newsrevenuehub/rev-engine
# [NRH-305] ThankYou page updates

DON’T MERGE ME IN TO DEVELOP UNTIL:


#### What's this PR do?
Adds some additional styles to the ThankYou page.
Adds some dynamic text.
Adds social share buttons.
Adds two new fields to rev program:
- `twitter_handle`
- `website_url`

#### How should this be manually tested?
Optionally add a `twitter_handle` and/or `website_url` to a rev program (and maybe even a `post_thank_you_redirect` url)
Make a donation for that rev program, and observe the content on the new ThankYou page. Check each social share button.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
